environment-pinned-linux.yml

```diff
@@ -17,24 +17,24 @@ dependencies:
   - async_generator=1.10=py_0
   - attrs=19.1.0=py_0
   - backcall=0.1.0=py_0
-  - bcftools=1.9=ha228f0b_4
+  - bcftools=1.8=h4da6232_3
```

Comment:
Hm, bcftools has somehow got downgraded in this update. I'm not too concerned about that as we rarely use bcftools, but it is unexpected.